newrelic/hooks/framework_graphql.py

```diff
@@ -16,14 +16,18 @@
 from newrelic.api.function_trace import FunctionTrace, FunctionTraceWrapper
 from newrelic.api.graphql_trace import GraphQLResolverTrace, GraphQLOperationTrace
 from newrelic.api.time_trace import notice_error, current_trace
-from newrelic.api.transaction import current_transaction
+from newrelic.api.transaction import current_transaction, ignore_transaction
 from newrelic.common.object_names import callable_name, parse_exc_info
 from newrelic.common.object_wrapper import function_wrapper, wrap_function_wrapper
 from newrelic.core.graphql_utils import graphql_statement
 from collections import deque
 from copy import copy
 
 
+GRAPHQL_IGNORED_FIELDS = frozenset(("id", "__typename"))
+GRAPHQL_INTROSPECTION_FIELDS = frozenset(("__schema", "__type"))
```

Comment:
According to [graphql docs](https://graphql.org/learn/introspection/), the list of fields for the introspection system is: __Schema, __Type, __TypeKind, __Field, __InputValue, __EnumValue, __Directive. Perhaps the additional fields should be added to the set to be cover all cases.

Reply:
The logic is identical to what the Node implementation is currently, and what Apollo server itself uses to identify these. I'm not exactly sure on the details as to why but the others can potentially be included in actual queries so we should probably not filter them out.